docs/learn/tour/tables.md

```diff
@@ -1,209 +1,307 @@
+| Segmented Table   | Partitioned tables distributed across multiple disks                                                 | When tables are larger than available disk size or access needs to parallelize |
+
+### Table Object
+
+In-memory objects in q can be persisted to disk as a single binary file, this is the simplest way that you can persist a table.
+
+Persisting both single binary table objects and splayed tables uses the keyword [`set`](../../ref/set.md) passing the file path to which the binary object will be persisted as the first argument. As a simple example
 
 ```q
-q)flip flip ec1
-city         country pop
------------------------------
-Istanbul     Turkey  15067724
-Moscow       Russia  12615279
-London       UK      9126366
-StPetersburg Russia  5383890
+q)table:([]id:neg[1000]?1000;power:1000?1f;voltage:1000?10f)
+q)`:object set table
+`:object
 ```
 
-So another way to construct a table:
+Exiting the process you can now check that the table has been persisted by listing the contents of your working folder
+
+=== "Linux/Mac"
+
+	```bash
+	$ ls
+	object
+	```
+
+=== "Windows"
+
+	```bash
+	$ dir
+	object
+	```
+
+Starting a new q proces you can then query the on-disk data by file reference
 
 ```q
-q)ec2:flip`city`country`pop!(`Berlin`Kyiv`Madrid;`Germany`Ukraine`Spain;3748148 3703100 3223334)
-q)ec2
-city   country pop
-----------------------
-Berlin Germany 3748148
-Kyiv   Ukraine 3703100
-Madrid Spain   3223334
+q)select from `:object where id>500
+id  power      voltage   
+-------------------------
+769 0.1870281  3.927524  
+591 0.6333324  3.017723  
+618 0.4418975  5.347096  
+645 0.5569152  0.8388858 
+541 0.3877172  1.959907
 ```
 
-CSVs are a common source of tables.
+### Splayed Table
 
-:fontawesome-regular-hand-point-right:
-[CSVs](csvs.md)
+Medium-sized tables (up to 100 million rows) and those with many columns are best stored on disk [splayed](https://en.wiktionary.org/wiki/splay "Wiktionary"): each column is stored as a separate file, rather than using a single file for the whole table. For example 
 
+```treeview
+splay/
+├── .d
+├── id
+├── power
+└── voltage
+```
 
-## Work
+The hidden file `.d` lists the columns in the order they appear in the table.
 
-There are two ways to work with tables and you can mix them to suit yourself. 
+The principle advantages of saving data in this way is that it allows for on-demand access to data in the columns required to perform a query. Limiting the required I/O for a query in this way allows us to deal with larger data volumes than binary tables or in-memory tables.
 
-QSQL queries are very like SQL. (Perhaps a little less verbose.)
+Persisting a splayed table is very similar to persisting a binary file, the key difference being that the supplied path should be a directory rather than a file
```

Comment:
```suggestion
Persisting a splayed table is very similar to persisting a binary file, the key difference being that the supplied path should be a directory rather than a file.
```